src/components/comments/dropdown-menu.tsx

```diff
@@ -61,10 +61,10 @@ export function DropdownMenu({
 
   return (
     <div
-      className={clsx(
-        'text-right bg-brand-50 py-3 pr-4 pl-2.5',
-        'shadow max-w-65 rounded-lg -mt-4'
-      )}
+      className={`
+        -mt-4 max-w-65 rounded-lg bg-brand-50
+        py-3 pr-4 pl-2.5 text-right shadow  
+      `}
```

Comment:
just to make sure I understand: did you forget the tw here or is it not necessary?

Reply:
In the comment of the helper it says it's just for use `outside of className` but you added it almost everywhere now. 
I'm confused :)

Reply:
I forgot it here, thanks for finding this.

Adding it everywhere improves the rendered dom.

the newlines around the template literals are added manually. I haven't investigated it - but there could be other plugins to automatically format it ...